### ISSUE TITLE
[ISSUE] build command-line-arguments: cannot load github.com/ugorji/go/codec: ambiguous import

### ISSUE THREAD

**Issue by vzool** [swaggo/gin-swagger#58]
Hi, I did do test one example [https://github.com/swaggo/swag/blob/master/example/celler/main.go](https://github.com/swaggo/swag/blob/master/example/celler/main.go) and I face this issue:
```
build command-line-arguments: cannot load github.com/ugorji/go/codec: ambiguous import: found github.com/ugorji/go/codec in multiple modules:
	github.com/ugorji/go v1.1.4 (/Users/vzool/Workspace/go/pkg/mod/github.com/ugorji/go@v1.1.4/codec)
	github.com/ugorji/go/codec v0.0.0-20190320090025-2dc34c0b8780 (/Users/vzool/Workspace/go/pkg/mod/github.com/ugorji/go/codec@v0.0.0-20190320090025-2dc34c0b8780)
```
How can I solve this?

Thanks

**Comment by bsdelf** [comment]
> Hi, I did do test one example https://github.com/swaggo/swag/blob/master/example/celler/main.go and I face this issue:
> 
> ```
> build command-line-arguments: cannot load github.com/ugorji/go/codec: ambiguous import: found github.com/ugorji/go/codec in multiple modules:
> 	github.com/ugorji/go v1.1.4 (/Users/vzool/Workspace/go/pkg/mod/github.com/ugorji/go@v1.1.4/codec)
> 	github.com/ugorji/go/codec v0.0.0-20190320090025-2dc34c0b8780 (/Users/vzool/Workspace/go/pkg/mod/github.com/ugorji/go/codec@v0.0.0-20190320090025-2dc34c0b8780)
> ```
> 
> How can I solve this?
> 
> Thanks

Add following line to your go.mod:

```
replace github.com/ugorji/go v1.1.4 => github.com/ugorji/go/codec v0.0.0-20190320090025-2dc34c0b8780
```

See discussion: https://github.com/gin-gonic/gin/issues/1673

**Comment by easonlin404** [comment]
fixed by #60
